### PR TITLE
Delete ARIA annotations page 

### PR DESCRIPTION
**Pull request by dawei-wang**
Delete ARIA annotations page per https://github.com/mdn/content/pull/30605#issuecomment-1830771776
Fix mdn/content#30605

### CODE REVIEW

**Comment by bsmth** [comment]
Hi @scottaohara 👋🏻 - I've had a look through this submission and read the related PR for context. I'm okay with removing the page; does your [+1 on the linked comment](https://github.com/mdn/content/pull/30605#issuecomment-1830771776) mean you also agree with these changes? Is there any information that could be moved over from the deleted document? Thanks :)

File: files/en-us/mozilla/firefox/releases/75/index.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ This article provides information about the changes in Firefox 75 that will affe
 
 ### Accessibility
 
-The new roles and objects related to [ARIA annotations](/en-US/docs/Web/Accessibility/ARIA/Annotations) are now exposed in Firefox, on Windows and Linux (bear in mind that these still won't be usable until screen readers start to support them):
+The new roles and objects related to [ARIA annotations](/en-US/docs/Web/Accessibility/ARIA) are now exposed in Firefox, on Windows and Linux (bear in mind that these still won't be usable until screen readers start to support them):
```
Comment (bsmth) [inline files/en-us/mozilla/firefox/releases/75/index.md:33]:
```suggestion
The new roles and objects related to [ARIA](/en-US/docs/Web/Accessibility/ARIA) are now exposed in Firefox, on Windows and Linux (bear in mind that these still won't be usable until screen readers start to support them):
```

File: files/en-us/web/accessibility/aria/roles/comment_role/index.md
Comment (bsmth) [inline files/en-us/web/accessibility/aria/roles/comment_role/index.md:1]: There's another mention on line 62 here:

>Nested comments are also possible with ARIA annotations — nest the comments inside one another, like so:

Which could be

>It's possible to nest comments inside one another, like so:

**Review by bsmth — APPROVED** [review]
Thanks a lot @dawei-wang.

I'd like to leave my +1 on these changes which would supersede https://github.com/mdn/content/pull/30605. It would be great to get another look from @aardrian before merging. The redirects and other changes look good (with some comments outstanding) 👍🏻

File: files/en-us/mozilla/firefox/releases/75/index.md (hunk 1 above)
Comment (scottaohara) [reply files/en-us/mozilla/firefox/releases/75/index.md:33]: just removing the word annotations makes this sentence read awkwardly.

Could be reworded to be:

"New [ARIA] roles and attributes are now exposed in...."

**Comment by aardrian** [comment]
I took a look. Bear in mind I am not familiar with the MDN process nor data structures, but at a glance this seems to gut the annotations language and I am happy with that.

File: files/en-us/mozilla/firefox/releases/75/index.md (hunk 1 above)
Comment (scottaohara) [reply files/en-us/mozilla/firefox/releases/75/index.md:33]: thanks

**Comment by bsmth** [comment]
Thanks, all! I'm going to merge now and close the other PR https://github.com/mdn/content/pull/30605

**Pull request merged**